panel/pane/markup.py

```diff
@@ -482,7 +483,8 @@ def _process_param_change(self, params):
             params['css_classes'] = ['markdown'] + params['css_classes']
         return super()._process_param_change(params)
 
-
+def _get_theme(config_theme)->str:
```

Comment:
Seems like this should be a classmethod.

Reply:
Changed